### PR TITLE
ramips: add support for Asus 4G-AX56

### PR DESCRIPTION
**Pull request by kddred**
rebase and fix up #11021 by Chuncheng Chen [ccchen1984@gmail.com](mailto:ccchen1984@gmail.com)

fix gpio values
fix uncompress error when booting
`Uncompressing Kernel Image ... lzma compressed: uncompress error 1`
`Must RESET board to recover`

Fibocom FG621-EA modem seems to be in upstream kernel as FM650-CN

### CODE REVIEW

File: target/linux/ramips/mt7621/base-files/etc/board.d/02_network [hunk 1]
```diff
@@ -182,7 +182,7 @@ ramips_setup_interfaces()
 	meig,slt866)
 		ucidef_set_interfaces_lan_wan "lan" "wan"
 		;;
-	*)
```
Comment (blogic) [inline target/linux/ramips/mt7621/base-files/etc/board.d/02_network:185]: this part looks wrong, please remove it, the patch should be good to merge once that change is made.

Comment (kddred) [reply target/linux/ramips/mt7621/base-files/etc/board.d/02_network:185]: oh, that does break things.

**Comment by blogic** [comment]
merged, Thanks !

**Comment by kddred** [comment]
those files shouldn't be executable?

**Pull request closed without merging**